
[2018-02-06 13:13:36] Traceback (most recent call last):
  File "d:\projects\hybrid-app-test\hybrid_app\app.py", line 98, in <module>
    main()
  File "d:\projects\hybrid-app-test\hybrid_app\app.py", line 29, in main
    HTML_code = get_output_html()
  File "d:\projects\hybrid-app-test\hybrid_app\app.py", line 42, in get_output_html
    with open(os.path.abspath('./output.html')) as f:
IOError: [Errno 2] No such file or directory: 'D:\\projects\\hybrid-app-test\\output.html'


[2018-02-06 16:23:24] Traceback (most recent call last):
  File "d:\projects\hybrid-app-test\hybrid_app\app.py", line 98, in <module>
    main()
  File "d:\projects\hybrid-app-test\hybrid_app\app.py", line 31, in main
    url=html_to_data_uri(HTML_code), window_title="your app name")
  File "d:\projects\hybrid-app-test\hybrid_app\app.py", line 48, in html_to_data_uri
    html = html.encode("utf-8", "replace")
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe2 in position 772263: ordinal not in range(128)


[2018-02-06 16:23:45] Traceback (most recent call last):
  File "d:\projects\hybrid-app-test\hybrid_app\app.py", line 98, in <module>
    main()
  File "d:\projects\hybrid-app-test\hybrid_app\app.py", line 31, in main
    url=html_to_data_uri(HTML_code), window_title="your app name")
  File "d:\projects\hybrid-app-test\hybrid_app\app.py", line 48, in html_to_data_uri
    html = html.encode("utf-8", "replace")
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe2 in position 772263: ordinal not in range(128)


[2018-02-06 16:39:45] Traceback (most recent call last):
  File "d:\projects\hybrid-app-test\hybrid_app\app.py", line 101, in <module>
    main()
  File "d:\projects\hybrid-app-test\hybrid_app\app.py", line 33, in main
    HTML_code = get_output_html()
  File "d:\projects\hybrid-app-test\hybrid_app\app.py", line 46, in get_output_html
    with open('output.html') as f:
IOError: [Errno 2] No such file or directory: 'output.html'


[2018-02-06 16:41:45] Traceback (most recent call last):
  File "d:\projects\hybrid-app-test\hybrid_app\app.py", line 101, in <module>
    main()
  File "d:\projects\hybrid-app-test\hybrid_app\app.py", line 33, in main
    HTML_code = get_output_html()
  File "d:\projects\hybrid-app-test\hybrid_app\app.py", line 46, in get_output_html
    with open('output.html') as f:
IOError: [Errno 2] No such file or directory: 'output.html'


[2018-02-06 16:42:11] Traceback (most recent call last):
  File "d:\projects\hybrid-app-test\hybrid_app\app.py", line 101, in <module>
    main()
  File "d:\projects\hybrid-app-test\hybrid_app\app.py", line 33, in main
    HTML_code = get_output_html()
  File "d:\projects\hybrid-app-test\hybrid_app\app.py", line 46, in get_output_html
    with open('./output.html') as f:
IOError: [Errno 2] No such file or directory: './output.html'


[2018-02-06 16:43:22] Traceback (most recent call last):
  File "d:\projects\hybrid-app-test\hybrid_app\app.py", line 101, in <module>
    main()
  File "d:\projects\hybrid-app-test\hybrid_app\app.py", line 33, in main
    HTML_code = get_output_html()
  File "d:\projects\hybrid-app-test\hybrid_app\app.py", line 46, in get_output_html
    with open('/output.html') as f:
IOError: [Errno 2] No such file or directory: '/output.html'


[2018-02-06 18:15:34] Traceback (most recent call last):
  File "d:\projects\hybrid-app-test\hybrid_app\app.py", line 105, in <module>
    main()
  File "d:\projects\hybrid-app-test\hybrid_app\app.py", line 33, in main
    HTML_code = get_output_html()
  File "d:\projects\hybrid-app-test\hybrid_app\app.py", line 50, in get_output_html
    with open('output.html') as f:
IOError: [Errno 2] No such file or directory: 'output.html'


[2018-02-06 18:15:55] Traceback (most recent call last):
  File "d:\projects\hybrid-app-test\hybrid_app\app.py", line 105, in <module>
    main()
  File "d:\projects\hybrid-app-test\hybrid_app\app.py", line 33, in main
    HTML_code = get_output_html()
  File "d:\projects\hybrid-app-test\hybrid_app\app.py", line 50, in get_output_html
    with open('output.html') as f:
IOError: [Errno 2] No such file or directory: 'output.html'


[2018-02-10 18:39:09] Traceback (most recent call last):
  File "d:\projects\hybrid-app-test\hybrid_app\app.py", line 104, in <module>
    main()
  File "d:\projects\hybrid-app-test\hybrid_app\app.py", line 29, in main
    HTML_code = get_output_html()
  File "d:\projects\hybrid-app-test\hybrid_app\app.py", line 49, in get_output_html
    with open(abs_html_path) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'D:\\projects\\hybrid-app-test\\output.html'

